tests/phpunit/helpers/class-wc-stripe-sdk-test-helper.php

```
@@ -0,0 +1,57 @@
+	public static function inject_sdk_payment_method_response( $data ) {
+		$data = (array) $data;
+		$obj  = \Stripe\Util\Util::convertToStripeObject( $data, [] );
+
+		$mock_service = new class( $obj ) {
+			private $response;
+
+			public function __construct( $response ) {
+				$this->response = $response;
+			}
+
+			public function __call( $name, $args ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)